db/seeds.rb

```diff
@@ -5,3 +5,29 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+admin = User.create(email: 'jason@admin.com', password: 'password', admin: true)
+
+jason =
+  User.create(
+    email: 'jason.ho@obf.ateneo.edu',
+    password: 'password',
+    admin: false,
+  )
+
+client =
+  IEX::Api::Client.new(
+    publishable_token: 'pk_714d506665774bd4911a522edde00754',
+    secret_token: 'sk_5a1c52d71b9a4d1d880b1f44646e44af',
+    endpoint: 'https://cloud.iexapis.com/v1',
```

Comment:
Your credential is confidential. Refrain from putting it in a plain text only.

Reply:
you can use `env` variables or the `credentials.yml.enc`

Reply:
noted, will work on this now